src/atom_vec_hybrid.cpp

```diff
@@ -235,8 +233,11 @@ void AtomVecHybrid::process_args(int narg, char **arg)
     styles_bonus = new AtomVec*[nstyles_bonus];
     nstyles_bonus = 0;
     for (int k = 0; k < nstyles; k++) {
-      if (styles[k]->bonus_flag)
+      if (styles[k]->bonus_flag) {
```

Comment:
Don't think this includes all the added lines I sent for this?
Should be one new comment line.  Also 2 size... = 0 lines inside the 
if (nstyles_bonus) {

Reply:
ok. but to avoid this, please next time send me the output of `git diff` or `diff -u oldfile.cpp newfile.cpp`. that makes these kinds of oversights infinitely less likely to happen.